src/statistics.jl

```diff
@@ -146,9 +146,9 @@ end
 # Take the highest lvl with at least 32 bins.
 # (Chose 32 based on https://doi.org/10.1119/1.3247985)
 function _select_lvl_for_std_error(B::LogBinner{N,T})::Int64 where {N, T}
-    B.count[1] == 0 && (return 0) # return 0 if no values have been pushed
+    B.count[1] == 0 && (return 1)           # results in NaN in std_error
```

Comment:
I'd replace `B.count[1] == 0` by `isempty(B)`, which is easier to parse.

Reply:
agreed